src/css/ds-button.css

```diff
@@ -2,7 +2,7 @@
   border-radius: 6px;
   padding: 0 12px;
   height: 40px;
-  font-size: 0.90625rem;
+  font-size: 1.8125rem;
```

Comment:
I don't think this is right, from the specs and using the --base-rem it should be `font-size: calc(16 / var(--rem-base) * 1rem);`

Reply:
Thanks! I will double check this.

Reply:
It has been fixed!